internal/client.go

```diff
@@ -626,6 +630,515 @@ type (
+		// expressions in CronExpressions will be translated to ScheduleSpec.Calendars on the server.
+		//
+		// For example, `0 12 * * MON-WED,FRI` is every M/Tu/W/F at noon, and is equivalent to this ScheduleCalendarSpec:
+		//
+		// client.ScheduleCalendarSpec{
+		// 		Second: client.ScheduleRanges{{}},
+		// 		Minute: client.ScheduleRanges{{}},
+		// 		Hour: client.ScheduleRanges{{
+		// 			Start: 12,
+		// 		}},
+		// 		DayOfMonth: client.ScheduleRanges{
+		// 			{
+		// 				Start: 1,
+		// 				End:   31,
+		// 			},
+		// 		},
+		// 		Month: client.ScheduleRanges{
+		// 			{
+		// 				Start: 1,
+		// 				End:   12,
+		// 			},
+		// 		},
+		// 		DayOfWeek: client.ScheduleRanges{
+		// 			{
+		// 				Start: 1,
+		//				End: 3,
+		// 			},
+		// 			{
+		// 				Start: 5,
+		// 			},
+		// 		},
+		// 	}
+		//
+		//
+		// The string can have 5, 6, or 7 fields, separated by spaces, and they are interpreted in the
+		// same way as a ScheduleCalendarSpec:
+		//	- 5 fields:         Minute, Hour, DayOfMonth, Month, DayOfWeek
+		//	- 6 fields:         Minute, Hour, DayOfMonth, Month, DayOfWeek, Year
+		//	- 7 fields: Second, Minute, Hour, DayOfMonth, Month, DayOfWeek, Year
+		//
+		// Notes:
+		//	- If Year is not given, it defaults to *.
+		//	- If Second is not given, it defaults to 0.
+		//	- Shorthands @yearly, @monthly, @weekly, @daily, and @hourly are also
+		//		accepted instead of the 5-7 time fields.
+		//	- @every <interval>[/<phase>] is accepted and gets compiled into an
+		//		IntervalSpec instead. <interval> and <phase> should be a decimal integer
+		//		with a unit suffix s, m, h, or d.
+		//	- Optionally, the string can be preceded by CRON_TZ=<timezone name> or
+		//		TZ=<timezone name>, which will get copied to ScheduleSpec.TimeZone. (In which case the ScheduleSpec.TimeZone field should be left empty.)
+		//	- Optionally, "#" followed by a comment can appear at the end of the string.
+		//	- Note that the special case that some cron implementations have for
+		//		treating DayOfMonth and DayOfWeek as "or" instead of "and" when both
+		//		are set is not implemented.
+		CronExpressions []string
+
+		// Skip - Any matching times will be skipped.
+		//
+		// All aspects of the schedulepb.ScheduleCalendarSpec—including seconds—must match a time for the time to be skipped.
+		Skip []ScheduleCalendarSpec
+
+		// StartAt - Any times before `startAt` will be skipped. Together, `startAt` and `endAt` make an inclusive interval.
+		// Optional: Defaulted to the beginning of time
+		StartAt *time.Time
+
+		// EndAt - Any times after `endAt` will be skipped.
+		// Optional: Defaulted to the end of time
+		EndAt *time.Time
+
+		// Jitter - All times will be incremented by a random value from 0 to this amount of jitter.
+		// Optional: Defaulted to 0
+		Jitter time.Duration
+
+		// Timezone - IANA timezone name, for example `US/Pacific`.
```

Comment:
not really a "name" if you're taking a `time.Location`. also, should it be a `*time.Location`? that's what `time.UTC` is and most `time` functions take/return

Reply:
Yeah this isn't a name, I think I missed this from the typescript API. I'll update the comment.  I'll update to take a pointer